contracts/container/contract.go

```diff
@@ -501,6 +592,47 @@ func Delete(containerID []byte, signature interop.Signature, token []byte) {
 	runtime.Notify("DeleteSuccess", containerID)
 }
 
+// Remove removes all data for the referenced container. Remove is no-op if
+// container does not exist. On success, Remove throws 'Removed' notification
+// event.
+//
+// See [Create] for details.
+func Remove(id []byte, invocScript, verifScript, sessionToken []byte) {
+	common.CheckAlphabetWitness()
+
+	if len(id) != interop.Hash256Len {
+		panic("invalid container ID length")
+	}
+
+	ctx := storage.GetContext()
+	cnrItemKey := append([]byte{containerKeyPrefix}, id...)
+	cnrItem := storage.Get(ctx, cnrItemKey)
+	if cnrItem == nil {
+		return
+	}
+
+	owner := ownerFromBinaryContainer(std.Deserialize(cnrItem.([]byte)).(Container).Value)
+
+	storage.Delete(ctx, cnrItemKey)
```

Comment:
Why not reusing `removeContainer` here?

Reply:
taking into account `Delete` is going to be removed, `removeContainer` is used once. I also noticed that when it's inlined, contract's NEF and manifest change (understandable but surprising still). And to me it's easier to see which storage items are affected w/o jumps. So i propose to keep as is, but dont mind to change